src/cs/utils/UtilTypes.cs

```diff
@@ -56,8 +56,9 @@ public struct Environment {
 	public int ImportedPollution; // The amount of pollution caused by energy imports
 
 	// Computes the value used to update the environment bar
-	public float EnvBarValue() =>
-		2.0f * (0.5f * Biodiversity - 0.5f * LandUse);
+	public double EnvBarValue() =>
+		//2.0f * (0.5f * Biodiversity - 0.5f * LandUse);
+		(Math.Exp(6.0f * (Biodiversity - 1.1f*LandUse))) / (Math.Exp(6.0f * (Biodiversity - 1.1f*LandUse))+1.0f);
```

Comment:
Could you add a comment explaining how you derived this ? I don't want any decision to be undocumented.